tests/test_fetch_same.py

```diff
@@ -0,0 +1,65 @@
+from nose.tools import assert_true, assert_false, assert_equal, \
+                        assert_list_equal, raises
+from . import PREFIX, CONN_INFO
+import datajoint as dj
+# import importlib
```

Comment:
removed commented code

Reply:
This is related to my debug setup. I have updated it to properly determine which import strategy is best.